src/evaluate.cpp

```diff
@@ -547,6 +547,8 @@ namespace {
   const int BishopCheck       = 588;
   const int KnightCheck       = 924;
 
+  // In Q8 fixed point
+  const int ThreeCheckKSFactors[3] = {820, 772, 599};
```

Comment:
I don't have a good name for those constants. `ThreeCheckKingSafetyFactorsIndexedByChecksRemainingMinusOne` is too long.